webapp/channels/src/components/admin_console/system_users/index.ts

```diff
@@ -34,7 +34,8 @@ function mapStateToProps(state: GlobalState) {
     const fromId = tableProperties?.cursorUserId ?? adminConsoleUserManagementTablePropertiesInitialState.cursorUserId;
     const fromColumnValue = tableProperties?.cursorColumnValue ?? adminConsoleUserManagementTablePropertiesInitialState.cursorColumnValue;
     const columnVisibility = tableProperties?.columnVisibility ?? adminConsoleUserManagementTablePropertiesInitialState.columnVisibility;
-    const searchTerm = tableProperties?.searchTerm ?? adminConsoleUserManagementTablePropertiesInitialState.searchTerm;
+    const searchTerm = tableProperties?.searchTerm;
```

Comment:
I've realized these columns are optional and don't need to be populated if they are `undefined`